rules-topics/04_combat.md

```diff
@@ -83,6 +106,21 @@ People wearing orange headbands are referred to as Pages.  Pages are people that
 
 **You may not ever physically strike a Page.**  Pages may not use combat skills, weapons, or throw packets.  If you engage in combat with a page, you may point your weapon or packet and say, "Page, I kill you."  They are treated as if they have 1 body.
 
+### Immunities
+
+---
+When you have an inherent ability or active effect that makes you explicitly immune to an effect you call "No Effect" if you are hit by a spell or weapon carrier using that effect.  If you are immune to part of the weapon damage call then you are immune to all of it.  The exception to this would Stasis or other abilities that explicitly state the things to which you become immune (and that call should always be, "No effect; Visibly in Stasis").
+
+Examples:
+* If you have Immune to Toxin, and you are hit by a Sleep dart, you would call "No Effect".
+* If you are Immune to Curse and got hit with a weapon swinging 10 Magic Curse, you would call "No Effect".
+* If you are Immune to Shock and got hit with an "Aether Shock Bolt 10" you would call "No Effect".
+* If you are not immune to Paralyze, and you get hit by a Paralyze spell, you would call "Got it". If you are then hit by another Paralyze effect, you would call "Got it" again.
+* If you are a PC with no non-standard effects, are not dead, and get hit by a Create Undead, you would call "Got it".
+* If you do not have Immune to Life on your stat card, are alive, and get hit by a Life spell, you would call "Got it".
+* If you are dead and get hit by a Death spell, you call "Got it".
+* If you are Dead, and you get hit by a Cure Light Wounds, Confine, Taint Blood, etc., you would call "Got it"
```

Comment:
Any reason we are referencing spells not in our system here?

Reply:
Because I am dumb.  Fixed